ui/src/app/App.scss

```diff
@@ -209,17 +186,17 @@ a {
     margin-bottom: 0;
     color: $aleph-greyed-text;
 
-    .bp3-tag {
+    .#{$bp-ns}-tag {
       @include rtlSupportInvertedProp(margin, left, $aleph-grid-size, null);
     }
 
     &__item {
       display: inline-block;
       min-width: 8em;
-      margin: $aleph-grid-size/2 0;
+      margin: $aleph-grid-size * 0.5 0;
```

Comment:
The `/` division syntax in SCSS has been deprecated as it is ambiguous and isn’t supported anymore by Dart Sass. The alternatives are using the `math.div(x, y)` function (which is more verbose), or multiplying in case of simple fractions.

(These changes have been applied automatically using a migrator script provided by the Sass maintainers -- it prefers multiplication over `math.div()` in simple cases like this.)

Reply:
Read more: https://sass-lang.com/documentation/breaking-changes/slash-div